modules/openshift-developer-cli-sample-applications.adoc

```diff
@@ -0,0 +1,114 @@
+[id='odo-sample-applications']
+= Sample applications
+include::openshift-developer-cli-attributes.adoc[]
+
+:context: odo-sample-applications
+
+{odo-title} is compatible with any language or run time listed within the OpenShift catalog of component types. In order to access the component over the web, you can create a URL using `odo url create`.
+
+Example:
+
+[source,sh]
+----
+NAME        PROJECT       TAGS
+dotnet      openshift     2.0,latest
+httpd       openshift     2.4,latest
+java        openshift     8,latest
+nginx       openshift     1.10,1.12,1.8,latest
+nodejs      openshift     0.10,4,6,8,latest
+perl        openshift     5.16,5.20,5.24,latest
+php         openshift     5.5,5.6,7.0,7.1,latest
+python      openshift     2.7,3.3,3.4,3.5,3.6,latest
+ruby        openshift     2.0,2.2,2.3,2.4,latest
+wildfly     openshift     10.0,10.1,8.1,9.0,latest
+----
+
+== Examples from Git repositories
+
+=== httpd
+
+This example helps build and serve static content via httpd on CentOS 7. For more information about using this builder image, including OpenShift considerations, see link:https://github.com/sclorg/httpd-container/blob/master/2.4/root/usr/share/container-scripts/httpd/README.md[here].
```

Comment:
IBM style guide says to avoid using "here" as link text. Suggest rewording so that a more appropriate term is used as the link text. Maybe something like: 

```
For more information about using this builder image, including OpenShift considerations, see the link:https://github.com/sclorg/httpd-container/blob/master/2.4/root/usr/share/container-scripts/httpd/README.md[Apache HTTP Server container image repository].
```

Same for all other example sections below.